larsim/EventGenerator/MultiPart/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+cet_build_plugin( MultiPartRain art::module
+        LIBRARIES
+            larcorealg::Geometry
```

Comment:
There are many libraries listed here.  I recommend going through them and making sure you keep only the ones you actually need.